src/workspacer.Shared/Layout/TallLayoutEngine.cs

```diff
@@ -96,5 +98,10 @@ private int GetNumInPrimary()
         {
             return _numInPrimary + _numInPrimaryOffset;
         }
+
+        private int MaybeReverse(int x, int windowsWidth, int spaceWidth)
```

Comment:
Could this be renamed to something like `CalcXPos`.

Reply:
I was struggling with that naming though, never though it can be that easy and clear.